src/sentry/api/endpoints/project_rules.py

```diff
@@ -20,6 +20,8 @@
 from sentry.signals import alert_rule_created
 from sentry.web.decorators import transaction_start
 
+MAX_RULES_PER_PROJECT = 100
```

Comment:
Can we make this configurable for Self hosted?

Reply:
I had that thought too, will do